ibis/backends/risingwave/__init__.py

```diff
@@ -109,6 +158,11 @@ def create_table(
         database: str | None = None,
         temp: bool = False,
         overwrite: bool = False,
+        # TODO(Kexiang): add `append only`
+        tbl_properties: dict | None = None,
+        data_format: str | None = None,
+        encode_format: str | None = None,
+        encode_properties: dict | None = None,
```

Comment:
Are all of these nouns the same ones used in the RisingWave documentation? If not, we should make them match and avoid introducing new nouns just for the Ibis API.

Reply:
It would be more natural, if we replace `tbl_properties` with `connector_properties`, but it will differentiate from Flink's corresponding nouns.

Reply:
Do you think it's appropreciate to rename it to `connector_properties`?

Reply:
If that matches the Risingwave terminology, then I think that is a good naming choice.